js/local-storage.js

```diff
@@ -0,0 +1,12 @@
+function saveData() {
+  let name = '';
+  let email = '';
+  let message = '';
```

Comment:
- [ ] great job. Please ensure that After filling the form, and not submitting it, the entered values should be saved in the browser. When the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data.